Release 1.2.0 (#101)
nautobot/nautobot-app-dns-models: docs/admin/release_notes/version_1.2.md

# v1.2 Release Notes

This document describes all new features and changes in the release. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Release Overview

- Major features or milestones

> - Changes to compatibility with Nautobot and/or other apps, libraries etc.

Reviewer: Can you replace these 2 lines with a one sentence summary of what's new in the release please?

# v1.2 Release Notes

This document describes all new features and changes in the release. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Release Overview

> - Made DNS models globally searchable.
> - Changed TTL of DNS Records to be optional and inherited from the zone.
> - Fixed issue that ARecord and AAAARecord models would accept any version of IP addresses.
> - Added DNS name length validation per RFD 1035.

Reviewer: ```suggestion
This release makes DNS models globally searchable in Nautobot and added DNS name length validation per RFC 1035.
```
